test/e2e/common/pods.go

```diff
@@ -631,7 +631,7 @@ var _ = framework.KubeDescribe("Pods", func() {
 					{
 						Name:    containerName,
 						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
-						Command: []string{"/bin/sh", "-c", "sleep 5", "/crash/missing"},
+						Command: []string{"/bin/sh", "-c", "sleep 15", "/crash/missing"},
```

Comment:
if `golang:latest` is 800mb, 15 seconds might not be enough for a non-flaky test.

Reply:
My bad. I was testing something and had this local change and I didn't realize I amended with this change.